idb/postgres/postgres.go

```diff
@@ -910,28 +913,31 @@ func (db *IndexerDb) CommitRoundAccounting(updates idb.RoundUpdates, round, rewa
 			if fs.AssetID == debugAsset {
 				db.log.Errorf("%d %s %s", round, b64(fs.Addr[:]), obs(fs))
 			}
-			_, err = fr.Exec(fs.Addr[:], fs.AssetID, fs.Frozen)
+			_, err = fr.Exec(fs.Addr[:], fs.AssetID, fs.Frozen, round)
 			if err != nil {
 				return fmt.Errorf("update asset freeze, %v", err)
 			}
 		}
 	}
 	if len(updates.AssetCloses) > 0 {
 		any = true
+		// TODO: What is acc doing? Seems to update the transaction that created the asset, but why?
```

Comment:
It makes asset rewind possible. Algos have ApplyData in the SignedTxnInBlock but asset data doesn't have the necessary data so we add it here. This records the pre-close balance so that rewind can restore it.

Reply:
Aha, thanks that solves it. I was puzzling over this one for a while. Is `aca` account close asset, and `aaamount` account asset amount?